core-web/libs/sdk/client/README.md

```diff
@@ -1,405 +1,624 @@
+    depth?: number;
+    languageId?: number;
+  }
+): Promise<DotCMSNavigationItem[]>;
+```
 
-const collectionResponse = await client.content
-    .getCollection('Blog') // Collection name
-    .limit(10) // Limit results to 10 items
-    .page(1) // Fetch the first page
-    .fetch(); // Execute the query
+#### Why Use `navigation.get()`?
 
-console.log(collectionResponse.contentlets);
-```
+-   Build dynamic menus and site trees with minimal configuration
+-   Avoid manual parsing or custom REST calls
+-   Support localized navigation out of the box
+-   Easily control navigation depth for responsive and nested UIs
 
-#### Sorting Content
+### client.content.getCollection(): Fetching Content Collections
 
-You can sort the content by any field in ascending or descending order:
+The `client.content.getCollection()` method allows you to query and retrieve a collection of content items of a specific type from dotCMS. It uses a builder pattern so you can fluently compose queries with filters, pagination, sorting, and more.
 
-```typescript
-const sortedResponse = await client.content
-    .getCollection('Blog')
-    .sortBy([{ field: 'title', order: 'asc' }]) // Sort by title in ascending order
-    .fetch();
-```
+#### Why Use `getCollection()`?
 
-#### Filtering by Language
+-   Query exactly the content you need
+-   Chain filters, sorting, and pagination cleanly
+-   Works great for lists, search results, or dynamic components
+-   Fully type-safe when used with TypeScript interfaces
 
-If you need to filter content by language, you can specify the `language` parameter:
+#### Basic Usage
 
-```typescript
-const languageFilteredResponse = await client.content
-    .getCollection('Blog')
-    .language(2) // Filter by language ID (e.g., 2)
-    .fetch();
+Here's how to fetch the first 10 items from the "Blog" content type:
+
+```ts
+const blogs = await client.content.getCollection('Blog').limit(10).page(1).fetch();
 ```
 
-#### Using Complex Queries
+#### Filtering and Querying Content
 
-You can build more complex queries using the query builder. For example, filter by `author` and `title`:
+You can apply query filters using a fluent builder pattern:
 
-```typescript
-const complexQueryResponse = await client.content
+```ts
+const filtered = await client.content
     .getCollection('Blog')
-    .query((qb) => qb.field('author').equals('John Doe').and().field('title').equals('Hello World'))
-    .fetch();
+    .query((qb) =>
+        qb.field('title').contains('dotCMS').and().field('publishDate').greaterThan('2023-01-01')
+    )
+    .limit(5)
+    .sortBy([{ field: 'publishDate', direction: 'desc' }]);
 ```
 
-#### Fetching Draft Content
+#### Search and Paginate Product Results by Title and Price
 
-To only fetch draft content, use the `draft()` method:
+```ts
+const searchResults = await client.content
+    .getCollection('Product')
+    .query((qb) => qb.field('title').contains('Book').and().field('price').greaterThan(25).build())
```

Comment:
`contains` and `greaterThan` are not valid method 

I think you meant `equals`